src/lib/trace/lttng/lttng.h

```diff
@@ -14,8 +14,11 @@
 #include <lttng/tracepoint.h>
 
 /* Map event to an LTTng tracepoint. */
-#define TOR_TRACE_LTTNG(subsystem, event_name, ...) \
-  tracepoint(subsystem, event_name, ## __VA_ARGS__);
+#define TOR_TRACE_LTTNG(subsystem, event_name, args...) \
+  tracepoint(tor_##subsystem, event_name, args)
```

Comment:
Is this new tor_##subsystem header a global identifier?  If so I would prefer if the prefix were something more tracing-specific:  Like, tor_trc_ instead of tor_.

Reply:
It is not within the code. It is with the event we export so consider this a bit of an "ABI" that is the tracing event the LTTng tracer will see here will be prefixes with "tor_circuit:<event_name>" for instance.

But apart from that, they don't become global symbol in the code afaik.